### ISSUE TITLE
Training setting for the tiny version

### ISSUE THREAD

**Issue by jingweizhang-xyz** [TerryPei/EfficientVMamba#29]
Hi,

Thank you for your great work! I currently working on a structure and want to apply my structure to your method. When I was reproducing the your tiny version. But found the accuracy is 74.7%. I believe it should be some hyper-parameter or training config issue.

Here is the training cfg:
AMP_OPT_LEVEL: ''
AUG:
  AUTO_AUGMENT: rand-m9-mstd0.5-inc1
  COLOR_JITTER: 0.4
  CUTMIX: 1.0
  CUTMIX_MINMAX: null
  MIXUP: 0.8
  MIXUP_MODE: batch
  MIXUP_PROB: 1.0
  MIXUP_SWITCH_PROB: 0.5
  RECOUNT: 1
  REMODE: pixel
  REPROB: 0.25
BASE:
- ''
DATA:
  BATCH_SIZE: 256
  CACHE_MODE: part
  DATASET: imagenet
  DATA_PATH: xxx
  IMG_SIZE: 224
  INTERPOLATION: bicubic
  MASK_PATCH_SIZE: 32
  MASK_RATIO: 0.6
  NUM_WORKERS: 8
  PIN_MEMORY: true
  ZIP_MODE: false
ENABLE_AMP: false
EVAL_MODE: false
FUSED_LAYERNORM: false
FUSED_WINDOW_PROCESS: false
LOCAL_RANK: 0
MODEL:
  DROP_PATH_RATE: 0.2
  DROP_RATE: 0.0
  LABEL_SMOOTHING: 0.1
  MMCKPT: false
  NAME: vssm_efficient
  NUM_CLASSES: 1000
  PRETRAINED: ''
  RESUME: ''
  TYPE: vssm_efficient
  VSSM:
    DEPTHS:
    - 2
    - 2
    - 4
    - 2
    DOWNSAMPLE: v1
    DT_RANK: auto
    D_STATE: 16
    EMBED_DIM: 48
    FLIP_LAYER: false
    FLIP_SCAN: false
    IN_CHANS: 3
    MLP_RATIO: 0.0
    PATCH_NORM: true
    PATCH_SIZE: 4
    SHARED_SSM: false
    SOFTMAX: false
    SSM_RATIO: 2.0
    WINDOWSIZE: 2
OUTPUT: xxx
PRINT_FREQ: 10
SAVE_FREQ: 1
SEED: 0
TAG: r1
TEST:
  CROP: true
  SEQUENTIAL: false
  SHUFFLE: false
THROUGHPUT_MODE: false
TRAIN:
  ACCUMULATION_STEPS: 1
  AUTO_RESUME: true
  BASE_LR: 0.001
  CLIP_GRAD: 5.0
  EPOCHS: 300
  LAYER_DECAY: 1.0
  LR_SCHEDULER:
    DECAY_EPOCHS: 30
    DECAY_RATE: 0.1
    GAMMA: 0.1
    MULTISTEPS: []
    NAME: cosine
    WARMUP_PREFIX: true
  MIN_LR: 1.0e-05
  MOE:
    SAVE_MASTER: false
  OPTIMIZER:
    BETAS:
    - 0.9
    - 0.999
    EPS: 1.0e-08
    MOMENTUM: 0.9
    NAME: adamw
  START_EPOCH: 0
  USE_CHECKPOINT: false
  WARMUP_EPOCHS: 20
  WARMUP_LR: 1.0e-06
  WEIGHT_DECAY: 0.05

INFO {"cfg": "configs/vssm/vssm_efficient_tiny.yaml", "opts": null, "batch_size": 256, "data_path": "/dev/shm/jingwezhang/imagenet", "zip": false, "cache_mode": "part", "pretrained": null, "resume": null, "accumulation_steps": null, "use_checkpoint": false, "disable_amp": false, "amp_opt_level": null, "output": "/lustre/nvwulf/home/jingwezhang/result/bivim/effvmamba/effvmamba_tiny", "tag": "r1", "eval": false, "throughput": false, "local_rank": 0, "fused_layernorm": false, "optim": null, "model_ema": true, "model_ema_decay": 0.9999, "model_ema_force_cpu": false}

We used 4 gpus with batch size  256 for each. Compared with the training logs you shared, our accuracy already fall behind at epoch 90 and the formatting is quite different from the log provided.

I can share the full training log if you need it.

Thank you very much!
